docs/src/quickstart.md

```diff
@@ -0,0 +1,90 @@
+# Quick Start
+
+## Installation
+
+After making sure that your ROCm stack is installed and working, simply add the required packages to your Julia environment:
+
+```julia
+]add HSARuntime, AMDGPUnative
+```
+
+If everything ran successfully, you can try loading the package and running the unit tests:
+
+```julia
+using AMDGPUnative
+]test AMDGPUnative
+```
+
+!!! warning
+    If while installing the package you get an error message along the lines of `GLIB_CXX_... not found`, it's possible that the C++ runtime used to build the ROCm stack and the one used by Julia are different.
+    If you have built the ROCm stack yourself or your system is slightly outdated this is very likely the case since Julia normally ships with its own C++ runtime.
```

Comment:
Just an FYI, I actually run into the opposite problem — Arch Linux's C++ ABI is ahead of that in Julia's (and package artifacts') builds, and since I have to build the ROCm from source, that newer version is what's required to run. The solution for me is to instead run `LD_PRELOAD=/usr/lib/libstdc++.so.6 julia`.

Reply:
Right, I think we had the exact same setup and the exact same issue.
Your fix probably makes more sense than my overkill "build everything from source" approach :sweat_smile: 

Updated the warning box accordingly.